flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/consumer/InputChannel.java

```diff
@@ -108,12 +102,17 @@ protected InputChannel(
 	// Properties
 	// ------------------------------------------------------------------------
 
-	int getChannelIndex() {
-		return channelIndex;
+	/**
+	 * Returns the index of this channel within its {@link InputGate}.
```

Comment:
nit: `{@link SingleInputGate}` probably seems more accurate. For `{@link InputGate}` we actually also have `UnionInputGate` instance.

Reply:
Good catch!